pro/src/components/pages/Bookings/PreFilters/FilterByBookingStatusPeriod/FilterByBookingStatusPeriod.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * @debt directory "Gaël: this file should be migrated within the new directory structure"
+ */
+
+import { addDays, subDays } from 'date-fns'
+import React, { FunctionComponent } from 'react'
+
+import PeriodSelector from 'components/layout/inputs/PeriodSelector/PeriodSelector'
+import { DEFAULT_BOOKING_PERIOD } from 'components/pages/Bookings/PreFilters/_constants'
+import { getToday } from 'utils/date'
+
+import FilterByStatus from './FilterByStatus'
+
+interface PreFiltersProp {
+  bookingBeginningDate: Date
+  bookingEndingDate: Date
+  bookingStatusFilter: string
+  offerEventDate: Date
+  offerVenueId: string
+}
+interface FilterByBookingStatusPeriodProps {
+  selectedBookingBeginningDate: Date
+  selectedBookingEndingDate: Date
+  selectedBookingFilter: string
+  updateFilters: (filters: Partial<PreFiltersProp>) => void
+}
+
+const FilterByBookingStatusPeriod: FunctionComponent<FilterByBookingStatusPeriodProps> =
+  ({
+    selectedBookingBeginningDate,
+    selectedBookingEndingDate,
+    selectedBookingFilter,
+    updateFilters,
+  }) => {
```

Comment:
Nous n'utilisons pas cette notation pour lui préférer : 

```suggestion
const FilterByBookingStatusPeriod =
  ({
    selectedBookingBeginningDate,
    selectedBookingEndingDate,
    selectedBookingFilter,
    updateFilters,
  }: FilterByBookingStatusPeriodProps): JSX.Element => {
```